ghost/core/core/shared/config/defaults.json

```diff
@@ -105,6 +105,9 @@
         "frontend": {
             "maxAge": 0
         },
+        "publicAsset": {
```

Comment:
I'd personally probably opt for plural here, idk something about `publicAsset` sounds wrong. We have plural on `customRedirects` but that's the only other example - not too fussed though

Reply:
Following the only example is better than introducing a new convention 👌